package.json

```diff
@@ -33,32 +33,27 @@
         "test": "jest"
     },
     "dependencies": {
-        "@lwc/compiler": "7.1.2",
-        "@lwc/engine-dom": "7.1.2",
-        "@lwc/engine-server": "7.1.2",
-        "@lwc/jest-preset": "16.1.0",
-        "@lwc/jest-resolver": "16.1.0",
-        "@lwc/jest-serializer": "16.1.0",
-        "@lwc/jest-transformer": "16.1.0",
-        "@lwc/module-resolver": "7.1.2",
-        "@lwc/synthetic-shadow": "7.1.2",
-        "@lwc/wire-service": "7.1.2",
-        "@salesforce/wire-service-jest-util": "4.1.4",
+        "@salesforce/wire-service-jest-util": "^4.1.4",
```

Comment:
Using a caret to avoid the need to keep bumping this project when the other one changes.